app/services/db/sp_return_log.rb

```diff
@@ -0,0 +1,39 @@
+module Db
+  class SpReturnLog
+    def self.update_user(request_id, user_id)
+      ::SpReturnLog.where(request_id: request_id).update_all(user_id: user_id)
+      nil
+    end
+
+    def self.add_return(request_id, user_id)
+      ::SpReturnLog.where(request_id: request_id).update_all(
```

Comment:
I know it's existing behavior, but does `update_all` skip AR callback, so it won't update the `updated_at` column? should we update that too while we're here?

Reply:
Yep, good catch. Added `updated_at`

Reply:
Turns out the table does not have `updated_at` or `created_at`